tasks/gene_typing/task_snippy_variants.wdl

```diff
@@ -50,6 +50,46 @@ task snippy_variants {
 
     # Compress output dir
     tar -cvzf "./~{samplename}_snippy_variants_outdir.tar" "./~{samplename}"
+
+    # compute number of reads aligned to reference
+    samtools view -c "~{samplename}/~{samplename}.bam" > READS_ALIGNED_TO_REFERENCE
+
+    # create coverage stats file
+    samtools coverage "~{samplename}/~{samplename}.bam" -o "~{samplename}/~{samplename}_coverage.tsv"
+
+    # capture number of variants from summary file
+    grep "VariantTotal" ~{samplename}/~{samplename}.txt | cut -f 2 > VARIANTS_TOTAL
+
+    # # compute proportion of reference genome with depth >= min_coverage
+    # compute read depth at every position in the genome
+    samtools depth -a "~{samplename}/~{samplename}.bam" -o "~{samplename}/~{samplename}_depth.tsv"
+
+    # if min_coverage is provided, use that value, otherwise use 10
+    if [[ ! -z "~{min_coverage}" ]]; then
```

Comment:
You could solve this by just defaulting min_coverage input to 10 in the inputs (`Int min_coverage = 10`)